widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt

```diff
@@ -261,47 +230,9 @@ internal class ActivityWatcherForCallVisualizerController(
         }
     }
 
-    @VisibleForTesting
-    internal fun onMediaUpgradeAccept(offer: MediaUpgradeOffer) {
-        Logger.d(TAG, "Media upgrade request accepted by visitor")
-        if (offer.video != null && offer.video != MediaDirection.NONE) {
-            watcher.openCallActivity()
-        } else {
-            Logger.e(TAG, "Audio upgrade offer in call visualizer", Exception("Audio upgrade offer in call visualizer"))
```

Comment:
I expected to see this log in a new place. However, looks like it's just about to be removed. Don't we need it?

Reply:
we're logging this event in `EngagementRepositoryImpl` line 449

```kotlin
private fun handleMediaUpgradeOffer(mediaUpgradeOffer: MediaUpgradeOffer) {
        Logger.d(TAG, "Media upgrade requested: $mediaUpgradeOffer")
        _mediaUpgradeOffer.onNext(mediaUpgradeOffer)
    }
``` 
I think it contains all the necessary information. What do you think about this matter?

Reply:
They are different, aren't they? The current one logs an error. The new one logs incoming offer.

Reply:
Yes, I somehow missed the 2-nd log.
That's a case that should not happen because we're displaying media upgrade dialogs based on these values' presence, so on a step when the request is accepted, those parameters should stay the same as before.

Reply:
Okay, got it.